ui/src/dashboards/utils/tableGraph.ts

```diff
@@ -14,6 +14,7 @@ import {
   DecimalPlaces,
 } from 'src/types/dashboards'
 import {TimeSeriesValue} from 'src/types/series'
+import {DataTypes} from 'src/shared/constants'
```

Comment:
total nitpick... but can we use the singular form for enums? e.g. `DataType` rather than `DataTypes`

Reply:
it looks like there is another enum already made that essentially does the same thing. ill just switch out this enum for that one.

Reply:
actually, that one that i mentioned wouldnt make as much sense. ill change DataTypes to DataType